packages/members-api/lib/services/member-bread.js

```diff
@@ -222,7 +222,7 @@ module.exports = class MemberBREADService {
                 await this.memberRepository.linkStripeCustomer({
                     customer_id: data.stripe_customer_id,
                     member_id: model.id
-                }, options);
```

Comment:
I think what we actually want is: {transacting: options.transacting} 🤔

But probably we want something like const sharedOptions = {transacting: options.transacting} and to update all of the places we pass options to use sharedOptions

Reply:
@allouis I tried this but my test hang if I make this change. 

This does work:
```
 const sharedOptions = options.transacting ? {
    transacting: options.transacting
} : {};
```

This does not work (two of my tests hang and fail due to a timeout):
```
const sharedOptions = {transacting: options.transacting}
```

This also does not work
```
const sharedOptions = {transacting: options.transacting ? options.transacting : null}
```

Weird that the transacting option is not simply ignored when it is null or undefined. 

I tried logging the options, which are:
```
{
  context: {
    internal: false,
    external: false,
    user: '1',
    api_key: null,
    integration: null,
    public: false
  },
  withRelated: [ 'labels' ]
}
```
So we can also decide to just pass `{}`. But I don't have the required knowledge about this part of the code to make that decision.

Reply:
Huh, how strange!

I like your current solution, it's future proof for if/when we force this to run in a transaction ☺️

Reply:
Thank you for the review! 😄